frontend/src/metabase/dashboard/components/QuestionPicker/QuestionPicker.tsx

```diff
@@ -84,24 +84,24 @@ function QuestionPickerInner({
         onChange={handleSearchTextChange}
       />
 
-      {(hasDataAccess || hasNativeWrite) && !isEmbeddingSdk && (
+      {(hasDataAccess || hasNativeWrite) && (
         <Flex gap="sm" mb="md" data-testid="new-button-bar">
           {hasDataAccess && (
             <Button
               variant="outline"
               className={S.newButton}
-              leftSection={<Icon name="insight" />}
-              onClick={() => onNewQuestion("notebook")}
+              leftSection={<Icon aria-hidden name="insight" />}
```

Comment:
This makes sure we can target the Element with `*ByRole("button", { name: "New Question" })`, otherwise, we'll also see the icon name in the accessible name which is bad for screenreaders.

Reply:
Yes, I tried that, and I don't remember how many errors I saw in the tests. So I only opt for one I'm working on instead. Let me create an RFC for this too, it's worth talking about it.